graphql/mappers/beta-banner-opt-out.js

```diff
@@ -13,6 +16,9 @@ export async function getBetaBannerContent() {
       bannerLink: resOptOutContent.scFragments[0].scLinkTextEn,
       bannerLinkHref: resOptOutContent.scFragments[0].scDestinationURLEn,
       bannerButtonText: resOptOutContent.scFragments[1].scLinkTextEn,
+      bannerButtonExternalText: resDictionaryContent.filter(
+        (entry) => entry.scId === 'opens-in-a-new-tab'
+      )[0].scTermEn,
```

Comment:
resDictionaryContent.find would be faster and easier to read

Reply:
Fixed :smiley: